protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.1.0
+
+* Introduce extension methods `update` and `copy` replacing `copyWith` and
+  `clone`.
+  Using these alternatives can result in smaller binaries, because it is defined
+  once instead of once per class.
+
+  One major gotcha is that if you import your proto files with a prefix these
+  methods will not appear on your instances.
+  Alternatively you can add
+
+  ```
+  import 'package:protobuf/protobuf.dart' show GeneratedMessageGenericExtensions;
+  ```
+
+  To get the extension methods in scope.
```

Comment:
In `protobuf/README.md` we say users typically do not need to import `protobuf`, so we should update it mentioning this extension method.